core/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterAllocationExplainAction.java

```diff
@@ -78,6 +78,7 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
                     client.admin().cluster().allocationExplain(req, new RestBuilderListener<ClusterAllocationExplainResponse>(channel) {
                 @Override
                 public RestResponse buildResponse(ClusterAllocationExplainResponse response, XContentBuilder builder) throws Exception {
+                    builder.humanReadable(true);
```

Comment:
Everywhere that we support the `human` flag, it defaults to `false`.  Even though this API will primarily consumed by humans, I think I'd prefer to keep the consistency of defaulting to `false`.  `remaining_delay_in_millis` is still quite readable for a human, and rerunning the request with `?human` is easy enough.